### DISCUSSION TITLE
Strapi Converting uploaded MP3 Audio in Media Library plug-in

### DISCUSSION THREAD

**Discussion by allenwstaffordjr** [strapi/strapi#5902] [Legacy Topics]
Hello,
I have had a great time working with Strapi. But I am having one problem the media library plugin keeps changing my uploaded mp3's to mpga's. How can I stop this?

**Comment by lauriejim** [comment]
Hello can you please give me more detail about your application.
The version of Strapi you are using, the database you are using and if you use an upload provider, wich one?
Thanks

**Reply by allenwstaffordjr** [reply]
I am using the digital droplet...
Strapi 3
Node 12
Yarm 1.22
Pm2  4.22.3
Nginx 1.16.1
Postgres 10
 everytime I upload an mp3 it converts it to an mpga.

**Reply by lauriejim** [reply]
Can you please precise full version number, please. especially for Strapi.

**Comment by derrickmehaffy** [comment]
Hello @allenwstaffordjr can you test in the latest version (v3.0.6) as I believe this version should fix the issue you noticed.

**Comment by mgarabedian** [comment]
I was having this same issue this week, upgrading to the latest version ( v3.1.3 ) solved it for me. Mp3 files with mime type _audio/mpeg_ are keeping their mp3 extensions now. Thanks!
